gapic-generator-java/src/main/java/com/google/api/generator/gapic/protoparser/Parser.java

```diff
@@ -91,15 +95,17 @@ public class Parser {
       ResourceName.createWildcard("*", "com.google.api.wildcard.placeholder");
 
   // Mirrors the sanitizer allowlist.
-  private static final Set<String> MIXIN_ALLOWLIST =
-      ImmutableSet.of(
-          "google.iam.v1.IAMPolicy",
-          "google.longrunning.Operations",
-          "google.cloud.location.Locations");
+  private static final Map<String, FileDescriptor> MIXIN_ALLOWLIST =
+      ImmutableMap.of(
+          "google.iam.v1.IAMPolicy", IamPolicyProto.getDescriptor(),
+          "google.longrunning.Operations", LocationsProto.getDescriptor(),
```

Comment:
Should this be `OperationsProto.getDescriptor()`?

Reply:
Yes, thanks for catching this.